content/SCALE/SCALEUIReference/Credentials/DirectoryServices/ActiveDirectory.md

```diff
@@ -63,20 +63,20 @@ The **Advanced Options** screen shows both the basic and advanced option setting
 |---------|-------------|
-| **Site Name** | Enter the relative distinguished name (RDN) of the site object in the AD server. This is the first component of the distingished name in AD. For more info, read [Configuring Active Directory]({{< relref "ConfigADSCALE.md" >}}). |
+| **Site Name** | Enter the relative distinguished name (RDN) of the site object in the AD server. This is the first component of the distinguished name in AD. For more info, read [Configuring Active Directory]({{< relref "ConfigADSCALE.md" >}}). |
 | **Kerberos Realm** | Select an existing realm from the dropdown list of options. Options are those configured in **Kerberos Realms**. After selecting **Enable (requires password or Kerberos principal)**, SCALE populates the **Kerberos Realm** and **Kerberos Principal** fields with what it discovered in AD. |
 | **Kerberos Principal** | Select the location of the principal in the keytab created in **Directory Services > Kerberos Keytabs**. After selecting **Enable (requires password or Kerberos principal)**, SCALE populates the **Kerberos Realm** and **Kerberos Principal** fields with what it discovered in AD. |
-| **Computer Account OU** | The organizational unit (OU) where new computer accounts are created. The OU string includes the distinguished name (DN) of the Computer Account OU that includes the hierarchical location of the OU within the directory structure. For example, *OU=Computers,DC=example,DC=com*. The OU string is read from top to bottom without relative distinguished names (RDNs). Slashes (/) are uses as delimiters, as in *Computers/Servers/NAS*. Backslashes (\) are used to escape characters but not as a separator. Backslashes are interpreted at multiple levels and might require doubling or even quadrupling to take effect. When this field is blank, new computer accounts are created in the Active Directory default OU. |
-| **AD Timeout** | Enter the number of seconds before timeout. To view the AD connection status, open the interface **Task Manager**, click **History** to open the **Jobs** screen. |
+| **Computer Account OU** | The organizational unit (OU) where new computer accounts are created. The OU string includes the distinguished name (DN) of the Computer Account OU that includes the hierarchical location of the OU within the directory structure. For example, *OU=Computers,DC=example,DC=com*. The OU string is read from top to bottom without relative distinguished names (RDNs). Slashes (/) are used as delimiters, as in *Computers/Servers/NAS*. Backslashes (\) are used to escape characters but not as a separator. Backslashes are interpreted at multiple levels and might require doubling or even quadrupling to take effect. When this field is blank, new computer accounts are created in the Active Directory default OU. |
+| **AD Timeout** | Enter the number of seconds before timeout. To view the AD connection status, open the interface go to **Task Manager > *History** to open the **Jobs** screen. |
```

Comment:
"open the interface go to **Task Manager > *History** to open the **Jobs** screen"

Confusing as described.

Reply:
reworded this in a new commit.